portfolio/src/main/java/com/google/sps/data/BlogHashMap.java

```diff
@@ -24,6 +24,7 @@
 import java.util.Set;
 
 public final class BlogHashMap {
+  private final String[] INTERNAL_TAGS = {"#general", "#wellbeing", "#music", "#games", "Tags"};
```

Comment:
Can you explain what "Tags" is for and why it doesn't follow the same pattern as the others? This is a perfect time to use a code comment. You might also consider making it a separate constant if it doesn't fit the pattern of INTERNAL_TAGS.